flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/inference/SubSequenceInputTypeStrategyTest.java

```diff
@@ -41,10 +44,10 @@
 			TestSpec
 				.forStrategy(
 					"A strategy used for IF ELSE with valid arguments",
-					InputTypeStrategies.startSequences()
+					InputTypeStrategies.compositeSequence()
 						.argument(logical(LogicalTypeRoot.BOOLEAN))
-					.subSequence(commonType(2))
-					.finish()
+						.subSequence(commonType(2))
```

Comment:
nit: `subsequence` instead of `subSequence` see https://en.wikipedia.org/wiki/Subsequence